Packs/CiscoFirepower/Integrations/CiscoFirepower/CiscoFirepower.py

```diff
@@ -1469,120 +3602,842 @@ def get_task_status_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict
+    If no ID is specified, retrieves list of all network analysis policies.
+    GET arguments: network_analysis_policy_id.
+    LIST arguments: expanded_response, limit, page, page_size.
+
+    Args:
+        client (Client): Session to Cisco Firepower Management Center to run desired requests.
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Returns:
+        CommandResults: Information about network analysis policies.
+    """
+    # GET arguments
+    network_analysis_policy_id = args.get('network_analysis_policy_id', '')
+    # LIST arguments
+    limit = arg_to_number(args.get('limit', 0))
+    page = arg_to_number(args.get('page', 0))
+    page_size = arg_to_number(args.get('page_size', 0))
+    expanded_response = argToBoolean(args.get('expanded_response', 'False'))
+
+    is_get_request = check_is_get_request(
+        get_args=[network_analysis_policy_id],
+        list_args=[expanded_response, limit, page, page_size]
+    )
+
+    if is_get_request:
+        raw_response = client.get_network_analysis_policy(
+            network_analysis_policy_id=network_analysis_policy_id,
+        )
+
+    else:  # is_list_request
+        raw_response, raw_responses = client.list_network_analysis_policy(
+            page=page,
+            page_size=page_size,
+            limit=limit,
+            expanded_response=expanded_response,
+        )
+
+    command_results = get_commandresults(
+        raw_response=raw_response,
+        command_headers_by_keys=NETWORK_ANALYSIS_POLICY_HEADERS_BY_KEYS,
+        command_title=f'Fetched {NETWORK_ANALYSIS_POLICY_TITLE}',
+        command_context=NETWORK_ANALYSIS_POLICY_CONTEXT,
+    )
+
+    if not is_get_request:
+        command_results.raw_response = raw_responses
+
+    return command_results
+
+
+def update_network_analysis_policy_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+        Modifies the network analysis policy associated with the specified ID.
+
+    Args:
+        client (Client): Session to Cisco Firepower Management Center to run desired requests.
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Raises:
+        ValueError: In case none of the update arguments were entered.
+
+    Returns:
+        CommandResults: Information about the modified network analysis policy.
+    """
+    network_analysis_policy_id = args['network_analysis_policy_id']
+    basepolicy_id = args.get('basepolicy_id', '')
+    name = args.get('name', '')
+    description = args.get('description')
+    inspection_mode = args.get('inspection_mode')
+    replicate_inspection_mode = arg_to_optional_bool(args.get('replicate_inspection_mode'))
+
+    update_arguments = (name, basepolicy_id, description, inspection_mode)
+
+    if not any(update_arguments):
+        raise ValueError(f'Please enter one of the update arguments: {update_arguments}')
```

Comment:
here it'd be a list of none, please use a string,
```suggestion
        raise ValueError(f'Please enter one of the update arguments: name, basepolicy_id, descrpition, inspection_mode')
```

Reply:
My bad 😅.
Changed it.